### PR TITLE
Publishing ZHE 4.0.0

### PR DESCRIPTION
**Pull request by cwarje**
Zenhub Enterprise 4.0.0 Release

- Zenhub platform
- Improved onboarding experience
- Board enhancements
- Integrations
- Productivity insights
- Security patches
- K3s, Redis, and Fluentd upgrades

### CODE REVIEW

File: k8s-cluster/base/non-dynamic/set-raptor-buckets-vars.yaml [hunk 1]
```diff
@@ -0,0 +1,376 @@
...
+  labels:
+    app.kubernetes.io/component: raptor-admin
+  name: raptor-admin
+spec:
+  template:
+    spec:
+      containers:
+        - name: raptor-admin
+          env:
+            # FIXME - Using this hack from the past until we implement a better way. See base/non-dynamic/set-toad-buckets-vars.yaml
```
Comment (m62534) [inline k8s-cluster/base/non-dynamic/set-raptor-buckets-vars.yaml:13]: we can omit this for this repo

File: k8s-cluster/base/non-dynamic/set-raptor-buckets-vars.yaml [hunk 2]
```diff
@@ -0,0 +1,376 @@
...
+            - name: S3_PRIVATE_SECRET_ACCESS_KEY
+              valueFrom:
+                secretKeyRef:
+                  key: bucket_secret_access_key # Using the existing files bucket that toad used before - file upload/download move to raptor is in progress on Cloud still
+                  name: configuration
+            - name: FILE_DOWNLOAD_URL
+              valueFrom:
+                configMapKeyRef:
+                  key: zenhub_server_address
+                  name: toad # Until we add this to the configuration secret - file upload/download move to raptor is in progress on Cloud still
```
Comment (m62534) [inline k8s-cluster/base/non-dynamic/set-raptor-buckets-vars.yaml:28]: can omit comments

File: k8s-cluster/base/non-dynamic/set-raptor-buckets-vars.yaml [hunk 3]
```diff
@@ -0,0 +1,376 @@
...
+            - name: IS_UPLOAD_FILE_TO_LOCAL
+              valueFrom:
+                configMapKeyRef:
+                  name: configuration
+                  key: local_files
+            # - name: FILES_UPLOAD_PATH # TODO - Determine what this needs to be with Backend team help
```
Comment (m62534) [inline k8s-cluster/base/non-dynamic/set-raptor-buckets-vars.yaml:34]: comment omit here too

File: k8s-cluster/base/non-dynamic/set-raptor-buckets-vars.yaml (hunk 1 above)
Comment (cwarje) [reply k8s-cluster/base/non-dynamic/set-raptor-buckets-vars.yaml]: Removed

File: k8s-cluster/base/non-dynamic/set-raptor-buckets-vars.yaml (hunk 2 above)
Comment (cwarje) [reply k8s-cluster/base/non-dynamic/set-raptor-buckets-vars.yaml]: Removed

File: k8s-cluster/base/non-dynamic/set-raptor-buckets-vars.yaml (hunk 3 above)
Comment (cwarje) [reply k8s-cluster/base/non-dynamic/set-raptor-buckets-vars.yaml]: Removed

**Review by m62534 — APPROVED** [review]

File: k8s-cluster/template-files/kraken-configmaps.template [hunk 4]
```diff
@@ -131,7 +131,7 @@ data:
       "isEnterprise": true,
       "isLicenseGovernanceEnabled": false,
       "isTrackerEnabled": false,
-      "isUploadFileToLocal": false,
+      "isUploadFileToLocal": true,
```
Comment (SeanEmac) [inline k8s-cluster/template-files/kraken-configmaps.template:134]:
```suggestion
      "isUploadFileToLocal": false,
```

**Review by SeanEmac — CHANGES_REQUESTED** [review]

**Review by SeanEmac — APPROVED** [review]
:rocket:

**Pull request merged**